flink-formats/flink-avro/src/main/java/org/apache/flink/formats/avro/typeutils/AvroSchemaConverter.java

```diff
@@ -255,28 +257,24 @@ private static DataType convertToDataType(Schema schema) {
 						decimalType.getScale())
 						.notNull();
 			}
-			return DataTypes.ARRAY(DataTypes.TINYINT().bridgedTo(Byte.class))
-					.notNull();
+			return DataTypes.ARRAY(DataTypes.TINYINT()).notNull();
```

Comment:
Actually, this should be `DataTypes.BYTES()` which would also match to Avro's `BYTES` type.

Reply:
or is Avro using `Byte[]` instead of `byte[]`

Reply:
Avro uses `ByteBuffer`, therefore we do a conversion there. I will change it to `DataTypes.BYTES()`.